nestkernel/event.h

```diff
@@ -582,18 +587,22 @@ class DataLoggingRequest : public Event
 inline DataLoggingRequest::DataLoggingRequest()
   : Event()
   , recording_interval_( Time::neg_inf() )
+  , recording_offset_( Time::ms( 0. ) )
```

Comment:
There's an `explicit ms(long t)`. I wonder why there's an ambiguity on the constructor. The long/double constructors should probably anyway be reduced to just the double constructor.